pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -229,6 +229,8 @@ type MachineConfigSpec struct {
 	OSImageURL string `json:"osImageURL"`
 	// Config is a Ignition Config object.
 	Config ignv2_2types.Config `json:"config"`
+
+	KernelArguments []string `json:"kernelArguments"`
```

Comment:
wondering if we need to update DeepCopy as well

Reply:
Just checked, our current `MachineConfig` deepcopy is implemented manually, but it already recurses on `Spec` so that covers all of this.